src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -388,6 +388,7 @@ kibana_vars=(
     xpack.security.authc.selector.enabled
     xpack.security.cookieName
     xpack.security.encryptionKey
+    xpack.security.experimental.fipsMode.enabled
```

Comment:
NIT: Thinking out loud there, but are we **sure** we want that under that **experimental** prefix instead of just flagging it as experimental **in the documentation**? 

experimental config prefixes means once the feature leaves its experimental state, the config property name changes too, meaning another config deprecation to move it, and overall more work and maintenance. 

So yeah, wouldn't documenting the flag as experimental be enough?

Reply:
I'm torn; I want to really make clear that this config option should not be used at the moment, but I do see what you mean more work/rework.

@legrego @azasypkin What do you think?

Reply:
I'm quite torn as well. We really do _not_ want this to be exposed/configured yet, and documenting it goes against that goal.

Reply:
Yeah, I think I'm in the "experimental-in-config-entry-name" camp mostly due to these reasons:

* We absolutely don't want anyone outside of Elastic to use or even know about it at this point, no usual "deprecation" flow is assumed. It's experimental in the broadest sense, it can theoretically lead to all sorts of consequences from unexpected downtime to data loss or violated security expectations.
* If anyone happens to figure out that such a setting exists and tries to use it before we can commit to supporting FIPS, well, that's on them. But when we're ready to officially support FIPS, we want to make sure they cannot upgrade before they go and read about the scope of FIPS support, expectations, and proper configuration that might potentially include other settings they will need to configure in addition to `fipsMode.enabled`.

Reply:
👍 fair enough, thanks for explaining the reasoning